### DISCUSSION TITLE
Disable key repeat

### DISCUSSION THREAD

**Discussion by tristengrant** [xremap/xremap#636] [Ideas]
Is it possible to disable key repeat? I'm remapping a numpad to use for shortcuts in Krita. If I map a key to, for example, V. It doesn't work correctly. The "v" key select a specific tool when its held down, but because they numpad it repeating, it doesn't work.

**Comment by k0kubun** [comment]
I'm not sure if it's directly supported, but did you try using `held`/`alone` as a workaround?

**Reply by tristengrant** [reply]
Thanks! I'll give this a try.
